Dockerfile

```diff
@@ -9,8 +9,6 @@ RUN yum install -y \
      libmnl \
      findutils \
      which
-RUN wget http://vault.centos.org/8.1.1911/BaseOS/x86_64/os/Packages/iproute-tc-4.18.0-15.el8.x86_64.rpm
```

Comment:
You don't really need to wget this rpm if you're going to install it from URL in the following line.